lua/harpoon/mark.lua

```diff
@@ -31,6 +31,25 @@ local function get_buf_name(id)
     return ""
 end
 
+local function mark_exists(buf_name)
+    local marks = harpoon.get_mark_config().marks
+
+    for idx = 1, #marks do
```

Comment:
I believe we have to use `M.get_length()` instead of `#marks` because of the index problem we discussed with prime.

Reply:
Oh, I didn't realize that. Thanks @asbjornhaland !

Reply:
Should be resolved now :-)